docs/components/concepts/access-control/mapping-rules.md

```diff
@@ -2,24 +2,30 @@
 id: mapping-rules
 title: "Mapping rules"
 sidebar_label: "Mapping rules"
-description: "Map your auth data to Camunda-specific data using mapping rules."
+description: "Map authentication entities from your Identity provider to Camunda-specific entities using mapping rules."
 ---
 
-:::info
-Mapping rules are only available for Camunda 8 Self-Managed with [OIDC-based authentication](/self-managed/installation-methods/helm/configure/connect-to-an-oidc-provider.md). They do not work with any other authentication methods, such as basic authentication.
+Mapping rules are used to dynamically manage access control in Camunda by [connecting your Identity Provider](connect-to-identity-provider.md) and assigning claims from a JWT access token to [Orchestration Cluster Identity](../../identity/identity-introduction.md) entities.
+
+In **Camunda 8 SaaS**, these assignments are automatically resolved based on your organization's setup.
```

Comment:
perhaps add that it is not possible for customers to use mapping rules in saas at this point